Dockerfile.lotus

```diff
@@ -56,19 +56,230 @@ COPY --from=builder /usr/lib/x86_64-linux-gnu/libOpenCL.so.1 /lib/
 RUN useradd -r -u 532 -U fc
 
 
+###
 FROM base AS lotus
 MAINTAINER Lotus Development Team
 
-COPY --from=builder /opt/filecoin/lotus      /usr/local/bin/
-COPY --from=builder /opt/filecoin/lotus-shed /usr/local/bin/
+COPY --from=builder /opt/filecoin/lotus /usr/local/bin/
+COPY scripts/docker-lotus-entrypoint.sh /
 
 ENV FILECOIN_PARAMETER_CACHE /var/tmp/filecoin-proof-parameters
 ENV LOTUS_PATH /var/lib/lotus
+ENV LOTUS_JAEGER_AGENT_HOST 127.0.0.1
+ENV LOTUS_JAEGER_AGENT_PORT 6831
+ENV DOCKER_LOTUS_IMPORT_SNAPSHOT https://fil-chain-snapshots-fallback.s3.amazonaws.com/mainnet/minimal_finality_stateroots_latest.car
+ENV DOCKER_LOTUS_WALLET_IMPORT ""
+
+RUN mkdir /var/lib/lotus /var/tmp/filecoin-proof-parameters
+RUN chown fc: /var/lib/lotus /var/tmp/filecoin-proof-parameters
+
+VOLUME /var/lib/lotus
+VOLUME /var/tmp/filecoin-proof-parameters
+
+USER fc
+
+EXPOSE 1234
 
-RUN mkdir /var/lib/lotus /var/tmp/filecoin-proof-parameters && chown fc /var/lib/lotus /var/tmp/filecoin-proof-parameters
+ENTRYPOINT ["/docker-lotus-entrypoint.sh"]
+
+CMD ["-help"]
```

Comment:
`/docker-lotus-entrypoint.sh` doesn't have a help flag

Reply:
> `/docker-lotus-entrypoint.sh` doesn't have a help flag

It only does because it exec's to lotus at the end.

Reply:
oh nice